templates/firefox-ios.xliff

```diff
@@ -2500,6 +2536,54 @@
         <source>Firefox Homepage</source>
         <note>In the settings menu, this is the title of the Firefox Homepage customization settings section</note>
       </trans-unit>
+      <trans-unit id="Settings.Home.Option.Wallpaper" xml:space="preserve">
+        <source>Wallpaper</source>
+        <note>In the settings menu, on the Firefox homepage customization section, this is the title for the option that allows users to access the wallpaper settings for the application.</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.Wallpaper.Accessibility.AmethystWallpaper" xml:space="preserve">
+        <source>Firefox wallpaper, amethyst pattern, button</source>
+        <note>In the settings menu, on the Firefox wallpaper customization screen, this is the accessibility string for the amethyst firefox wallpaper.</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.Wallpaper.Accessibility.CeruleanWallpaper" xml:space="preserve">
+        <source>Firefox wallpaper, cerulean pattern, button</source>
+        <note>In the settings menu, on the Firefox wallpaper customization screen, this is the title accessibility string for the cerulean firefox wallpaper.</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.Wallpaper.Accessibility.DefaultWallpaper" xml:space="preserve">
+        <source>Default clear wallpaper, button</source>
+        <note>In the settings menu, on the Firefox wallpaper customization screen, this is the accessibility string for the default wallpaper.</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.Wallpaper.Accessibility.SunriseWallpaper" xml:space="preserve">
+        <source>Firefox wallpaper, sunrise pattern, button</source>
+        <note>In the settings menu, on the Firefox wallpaper customization screen, this is the title accessibility string for the sunrise firefox wallpaper.</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.Wallpaper.Accessibility.ToggleButton" xml:space="preserve">
+        <source>Homepage wallpaper cycle toggle</source>
+        <note>In the settings menu, on the Firefox wallpaper customization screen, this is the accessibility string of the toggle for turning wallpaper cycling shortcut on or off on the homepage.</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.Wallpaper.CollectionTitle" xml:space="preserve">
+        <source>OPENING SCREEN</source>
```

Comment:
Just want to make extra sure this should indeed be all caps :) 
I'm not sure about the comment either... is the title indeed "OPENING SCREEN" here?

Reply:
Confirmed all caps. :)

Reply:
Thanks Roux! Then this can stay as is